conversion/conversion.go

```diff
@@ -187,6 +187,18 @@ func performSnapshotMigration(config writer.BatchWriterConfig, conv *internal.Co
 	return batchWriter, nil
```

Comment:
Why does this method have err in return params but only returns nil? Am I missing something here ?

Reply:
Looks like a refactoring we missed [here](https://github.com/cloudspannerecosystem/harbourbridge/pull/323) in this PR, we can remove the error being returned.

Reply:
Fixed